spec/system/publishers/publishers_can_manage_job_applications_for_a_vacancy_spec.rb

```diff
@@ -75,7 +75,9 @@
         find(".govuk-checkboxes__item").click
       end
       click_on I18n.t("publishers.vacancies.job_applications.candidates.update_application_status")
-      find(".govuk-tag--purple").click
+      # wait for page load
+      find(".govuk-radios")
```

Comment:
[Minor] I think it might be better to use expect(page).to have_css(".govuk-radios"), I think the waiting behaviour is the same but might be a bit easier to understand at a glance than finding an element but not interacting with it. Also the assertion failing might make the issue with the test a bit more obvious.